scss/_layout_grid.scss

```diff
@@ -11,3 +11,21 @@ $shelves-column-name: $grid-col-name;
 @import
 'grid/shelves',
 'grid/shelves-grid';
+
+// Remove double margin bottoms on rows
+.row >,
+[class^="col-"] > {
+  p,
+  ul,
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6,
+  table {
```

Comment:
Could we use `*` here instead of elements? There maybe be many more like article, div, etc.